<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>602</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>710</width>
    <height>602</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>710</width>
    <height>602</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Netease Music Qt</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="MusicAvater">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="text">
     <string>MusicAvater</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Music_Label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>481</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>ÂæÆËΩØÈõÖÈªë</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MusicName</string>
    </property>
   </widget>
   <widget class="QLabel" name="AlbumName_Label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>ÂæÆËΩØÈõÖÈªë</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>AlbumName:</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ArtistName_Label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>ÂæÆËΩØÈõÖÈªë</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ArtistName:</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>391</width>
      <height>311</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>389</width>
       <height>309</height>
      </rect>
     </property>
     <widget class="QListWidget" name="LyricList">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>391</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="Like_Button">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>430</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>üß°ÂñúÊ¨¢</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Download_Button">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>430</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="text">
     <string>‰∏ãËΩΩ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Share_Button">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>430</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ÂàÜ‰∫´</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="CommentWindowButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>550</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ËØÑËÆ∫</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="PlayButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>530</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Resources/play.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="PreButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Resources/skip_backward.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="NextButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>540</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Resources/ward.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="ProgressSlider">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>560</y>
      <width>511</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="sliderPosition">
     <number>0</number>
    </property>
    <property name="tracking">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Small_Music_Artist">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>540</y>
      <width>391</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Music-Artist</string>
    </property>
   </widget>
   <widget class="QListWidget" name="PlayList">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>260</y>
      <width>221</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="SearchWindowButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>520</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ÊêúÁ¥¢</string>
    </property>
    <property name="icon">
     <iconset resource="Resources/imagereousrce.qrc">
      <normaloff>:/Resources/doc_search.png</normaloff>:/Resources/doc_search.png</iconset>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="SettingWindowButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>460</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ËÆæÁΩÆ</string>
    </property>
    <property name="icon">
     <iconset resource="Resources/imagereousrce.qrc">
      <normaloff>:/Resources/list_expand.png</normaloff>:/Resources/list_expand.png</iconset>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="TimeLabel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>540</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>240</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Êí≠ÊîæÂàóË°®:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowOriginLyric">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>460</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ÊòæÁ§∫ÂéüÊ≠åËØç</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowTranslatedLyric">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>460</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ÊòæÁ§∫ÁøªËØë</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resources/imagereousrce.qrc"/>
 </resources>
 <connections/>
</ui>
